### ISSUE TITLE
Losing team should cease to spawn in Point Match (optional setting)

### ISSUE THREAD

**Issue by TheGermanMason** [RF-Stuff/Bugs-Suggestions#127]
I like how the enemy team stops spawning in when you win in Battalion, though, I have a few issues with Battalion which is why I usually prefer Point Matches since the amount of bots don't change, literally every flag matters (Unlike Battalion where it's only important to have enough flags to support the battalions you have) and both teams can still turn the match around even if they had a bad start.

Long story short, I like Point Matches, but I'd like if I could fully wipe the map once I've been victorious.
